nsjail.h

```diff
@@ -157,6 +158,7 @@ struct nsjconf_t {
 	std::vector<int> openfds;
 	std::vector<int> caps;
 	std::vector<std::string> ifaces;
+	std::vector<std::pair<int, int>> pipes;
```

Comment:
Needs to be added to config proto, CLI arguments and documented

Reply:
Not really a config option as mentioned above.